### PR TITLE
Downgrade storybook server; Bump other storybook packages

### PR DESCRIPTION
**Pull request by JeffGuKang**
## Description

Change storybook package versions. 
`@storybook/react-native-server` had a bug that cannot use addon tab on browser as https://github.com/storybookjs/react-native/issues/24.

## Related Issues

https://github.com/storybookjs/react-native/issues/24

## Tests

Manual check

1. `yarn start`
2. `yarn storybook`
3. `yarn ios`

image

 

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui-native/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [ ] I signed the [CLA].
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Comment by hyochan** [comment]
@JeffGuKang Could you tell how this is going? Is this still needed?

**Comment by JeffGuKang** [comment]
@hyochan There seems to be no update yet. I will check continually. 
And storybook repo for react native was split from original.

**Comment by TfADrama** [comment]
is this pr dead?

**Comment by JeffGuKang** [comment]
@hyochan @TfADrama  There is no update from https://github.com/storybookjs/react-native/issues/24 yet. 
This PR needs if we want to use storybook control on the web browser. But we can use on the device continually like now without this PR.

File: .vscode/settings.json [hunk 1]
```diff
@@ -23,5 +23,5 @@
   },
   "javascript.preferences.importModuleSpecifier": "relative",
   "typescript.preferences.importModuleSpecifier": "relative",
-  "prettier.configPath": ".prettierrc.js"
-}
+  "prettier.configPath": ".prettierrc.js",
```
Comment (hyochan) [inline .vscode/settings.json:26]: Good to not put trailing comma in pure json

**Comment by JeffGuKang** [comment]
I will check the Storybook and open PR later to solve this issue.

**Pull request closed without merging**